config/kube/crds/namespaced/networking.k8s.io_ingresses.yaml

```diff
@@ -0,0 +1,366 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kcp-dev/kubernetes/pull/4
```

Comment:
(1) Don't we need this in the CRDs too?

(2) Any URL will do (as noted in https://github.com/MikeSpreitzer/kcp-edge-mc/commit/b75c1e3d47134edac111acd47b0e6d830f24b489).

(3) What is going on here is not related to kcp, so I would not use a kcp URL.  As you see in (2), elsewhere I have used https://github.com/kubernetes/enhancements/pull/1111 (which is the PR that introduced the design change requiring this annotation, thus has explanatory power).

Reply:
Changed the annotation value for `networking.k8s.io_ingresses`.